Installers/default.build

```diff
@@ -40,6 +40,8 @@
 !define INSTALLERVERSION "${buildNumber}"
         </echo>
         <call target="download"/>
+        <exec program="C:\Windows\system32\cmd.exe" commandline='/C rmdir /S /Q WindowsUniversal' workingdir="${project::get-base-directory()}\..\MonoGame.Framework\bin" failonerror='false' />
+		<exec program="C:\Windows\system32\cmd.exe" commandline='/C rename WindowsUAP WindowsUniversal' workingdir="${project::get-base-directory()}\..\MonoGame.Framework\bin" />
```

Comment:
This will likely bite us some time in the future.  It would have been better to just not have done this and let the .nsi script pull from the existing WindowsUAP folder.